File: fix_import.py
import re

with open('src/components/store-manager/StoreProAccount.tsx', 'r') as f:
    code = f.read()

code = code.replace('} , Volume2 } from "lucide-react";', ', Volume2 } from "lucide-react";')

with open('src/components/store-manager/StoreProAccount.tsx', 'w') as f:
    f.write(code)

File: fix_import.js
const fs = require('fs');
let code = fs.readFileSync('src/components/store-manager/StoreProAccount.tsx', 'utf8');

code = code.replace(/\} , Volume2 \} from "lucide-react";/g, ', Volume2 } from "lucide-react";');

fs.writeFileSync('src/components/store-manager/StoreProAccount.tsx', code);
